packages/admin/cms-admin/src/blocks/iframebridge/IFrameBridge.tsx

```diff
@@ -154,10 +154,10 @@ export const IFrameBridgeProvider = ({ children, onReceiveMessage }: PropsWithCh
                                 const message: IAdminHoverComponentMessage = { cometType: AdminMessageType.HoverComponent, data: { adminRoute } };
                                 sendMessage(message);
                             },
-                            sendContentScope: (contentScope) => {
+                            sendContentScope: (contentScope, encryptedContentScope) => {
                                 const message: IAdminContentScopeMessage = {
                                     cometType: AdminMessageType.ContentScope,
-                                    data: { contentScope },
+                                    data: { contentScope, encryptedContentScope },
```

Comment:
strictly speaking it is not encrypted, it's "only" signed

and can't we decode the scope from the JWT on the client (without verify) instead to passing it plain AND as JWT?